Off-Top-App/off-top-java
# OT-98 | Properly convert audio data into a audio file in Spring-Boot

**What I Did:**
- Created new services to be domain-specific and make handling our WebSocket more readable
- Successfully wrote byte data into an audio wave file
- Am able to send the byte data properly through my Kafka Producer and can even read it in my Python Kafka Consumer
**How To Test**
- There is a file in the spring boot directory called AudioFile.wav, you can click on it and see that it is playable
- Also you can see that our message that get's send from our WebSocket and then produced into Kafka is being sent properly when utilizing the front-end applciation
